### ISSUE TITLE
Examples/Docs

### ISSUE THREAD

**Issue by ggVGc** [jobjo/HJScript#2]
Is there anywhere to see any kind of usage example of this? As a new user it looks quite interesting, but it's almost impossible to start out without any direction whatsoever on how a normal HJScript program is structured.
Even just a very simple hello world example in the repo would be very helpful.

**Comment by jobjo** [comment]
Sorry but this is not an active project. For generating JavaScript from Haskell there are numerous options, see this page for a summary: https://wiki.haskell.org/The_JavaScript_Problem

**Comment by ggVGc** [comment]
Yeah, I understand it's no longer maintained, and in fact my main wish was to use this project as a base for something similar, or fork it and continue maintaining it. But before that I wanted to get it up and running, hence looking for any sample. I am aware of the other JS alternatives that exist for Haskell, but I don't think there exists one currently that is what HJScript is? JMacro is similar but is not an EDSL in the way HJScript is, unless I am mistaken. What is the closest current alternative for HJScript in your opinion?
